contrib/Dockerfile.builder.fedora

```diff
@@ -0,0 +1,37 @@
+FROM fedora:27
+
+ENV BITCOIN_VERSION 0.16.0
+WORKDIR /tmp
+
+RUN dnf update -y && \
+	dnf groupinstall -y \
+		'C Development Tools and Libraries' \
+		'Development Tools' && \
+	dnf install -y \
+		clang \
+		gmp-devel \
+		libsq3-devel \
+		python2-devel \
+		python3-devel \
+		python3-pip \
+		python3-setuptools \
+		net-tools \
+		libsodium-devel \
+		net-tools \
+		valgrind \
+		wget && \
+	dnf clean all && \
+	git clone https://github.com/ElementsProject/lightning.git && \
```

Comment:
@cdecker actually, I have added a git clone here for lightning.git which don't have in Dockerfile.builder. Should I remove it?